core/vm/gas_table_test.go

```diff
@@ -105,3 +105,50 @@ func TestEIP2200(t *testing.T) {
 		}
 	}
 }
+
+var createGasTests = []struct {
+	code    string
+	eip3860 bool
```

Comment:
Seems to be unused.

Reply:
Used on line 139

Reply:
Thanks 🙏🏻

Reply:
Sorry, I skimmed through that this is a struct and not a function.